src/components/RateAgentModal/index.tsx

```diff
@@ -80,7 +86,19 @@ export default function RateAgentModal({
           <h4>
             {t('additionalComments') + ' ' + t('optional', { ns: 'shared' })}
           </h4>
-          <p>{t('additionalCommentsHelpText')}</p>
+          <p>
+            {t('additionalCommentsHelpText')}
+            <OverlayTrigger
+              placement="right"
+              overlay={
+                <Popover>
+                  <Popover.Body>{t('additionalCommentsTooltip')}</Popover.Body>
+                </Popover>
+              }
+            >
+              <i className="bi bi-info-circle"></i>
```

Comment:
Lets add `ms-1` to these class names to give it a little breathing room on the left.